### ISSUE TITLE
MissingMethodException with Gradle 3.0

### ISSUE THREAD

**Issue by gordontyler** [maiflai/gradle-scalatest#38]
When running the test task using Gradle 3.0, it fails with the following exception:

```
groovy.lang.MissingMethodException: No signature of method: org.gradle.StartParameter.isColorOutput() is applicable for argument types: () values: []
        at com.github.maiflai.ScalaTestAction.color(ScalaTestAction.groovy:101)
        at com.github.maiflai.ScalaTestAction$color$3.callStatic(Unknown Source)
        at com.github.maiflai.ScalaTestAction.reporting(ScalaTestAction.groovy:123)
        at com.github.maiflai.ScalaTestAction$reporting$0.callStatic(Unknown Source)
        at com.github.maiflai.ScalaTestAction.getArgs(ScalaTestAction.groovy:129)
        at com.github.maiflai.ScalaTestAction.makeAction(ScalaTestAction.groovy:70)
        at com.github.maiflai.ScalaTestAction$makeAction.callStatic(Unknown Source)
        at com.github.maiflai.ScalaTestAction.execute(ScalaTestAction.groovy:29)
        at com.github.maiflai.ScalaTestAction.execute(ScalaTestAction.groovy)
        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:623)
        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:606)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:80)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:61)
```

**Comment by maiflai** [comment]
Hi,

I think this is resolved in the current release (0.14).

Please could you confirm the version of the plugin that you are using?

Thanks,
Stu

**Comment by gordontyler** [comment]
Thanks, using 0.14 does resolve the problem. Sorry, I should have checked for a new release first. :)
